internal/syncerror/known_sync_error_windows.go

```diff
@@ -17,12 +17,16 @@
 
 package syncerror
 
-import "golang.org/x/sys/windows"
+import (
+	"errors"
+
+	"golang.org/x/sys/windows"
+)
 
 // knownSyncError returns true if the given error is one of the known
 // non-actionable errors returned by Sync on Windows:
 //
 // - sync /dev/stderr: The handle is invalid.
 func KnownSyncError(err error) bool {
-	return err == windows.ERROR_INVALID_HANDLE
+	return errors.Is(err, windows.ERROR_INVALID_HANDLE)
```

Comment:
```
internal\syncerror\known_sync_error_windows.go:27:9: comparing with == will fail on wrapped errors. Use errors.Is to check for a specific error (errorlint)
        return err == windows.ERROR_INVALID_HANDLE
```
Interesting, didn't know golangci-lint doesn't check go files for Windows when ran on unix systems (ci).